lib/eco/treemanager.py

```diff
@@ -289,9 +290,9 @@ def __init__(self):
         self.savenextparse = False
         self.saved_lines = {}
         self.saved_parsers = {}
-        self.profile_data = {} # line no -> (float, node) (raw data)
-        self.profile_map = {}  # node -> str
-        self.profile_is_dirty = False
+
+        self.tool_information_is_dirty = False
```

Comment:
Can we maybe use a shorter name than `tool_information_is_dirty`? Maybe `tool_data_is_dirty` or something like that?